databricks/koalas/missing/indexes.py

```diff
@@ -225,6 +223,16 @@ class _MissingPandasLikeMultiIndex(object):
     # Functions we won't support.
     values = common.values(unsupported_property)
     array = common.array(unsupported_property)
+    codes = unsupported_property(
+        'codes',
+        reason="'codes' requires to collect all data into the driver which is against the "
+               "design principle of Koalas. Alternatively, you could call 'to_pandas()' and"
+               " use 'codes' property in pandas.")
+    levels = unsupported_property(
+        'levels',
+        reason="'levels' requires to collect all data into the driver which is against the "
+               "design principle of Koalas. Alternatively, you could call 'to_pandas()' and"
+               " use 'levels' property in pandas.")
```

Comment:
@ueshin let me merge this one for now .. but we can always flip this decision if I missed anything.